### ISSUE TITLE
make plots of USDA data

### ISSUE THREAD

**Issue by lizzieinvancouver** [lizzieinvancouver/egret#54]
See if we can plot: 

- chilling versus % or other response variables
- germination C versus % or other response variables

**Comment by lizzieinvancouver** [comment]
@wangxm-forest Can you work on this when done with USDA cleaning?

**Comment by wangxm-forest** [comment]
@lizzieinvancouver Yes, I will work on this once I finish cleaning the code (which is almost done)!

**Comment by wangxm-forest** [comment]
@lizzieinvancouver 
After cleaning, we have the chilling duration and chilling (Y/N) columns. Would you like me to plot chilling duration vs. the response variables?

We don’t have a specific germination temperature column in the USDA dataset, but we do have 'tempDay' and 'tempNight,' which I assume refer to germination temperatures. Have we made any decisions on how to proceed with this? Should we calculate the mean, or should we plot them separately?

**Comment by lizzieinvancouver** [comment]
@wangxm-forest Great questions! I would make some plots that seem useful to you to start and then bring up this question Monday at the retreat.

**Comment by wangxm-forest** [comment]
We decided to average the tempDay and tempNight as the germTemp.

**Comment by lizzieinvancouver** [comment]
@dbuona Was going to think on modeling this some today (and other stuff) so he might have plots to add here.

**Comment by dbuona** [comment]
@wangxm-forest and @lizzieinvancouver in source("scrapeUSDAseedmanual/cleaning/source/cleanResponseUsda.R") of the main USDAT cleaning script the data goes from having 618 rows to 6,180 rows, and alot of them are NAs. @wangxm-forest can you check this script and see if you can figure out why it is happening?

**Comment by wangxm-forest** [comment]
@dbuona 
I think it is due to following code, which converts several columns of responseVar to a single Column, but it repeats all the 10 types of responseVar for all the lines. I didn't notice it until you pointed this out, I'll see if there's better wat to do this.

`d <- d %>%
  group_by(species_name) %>%
  pivot_longer(cols = c("germination_time_days","total_germination_percent","avg_germination_percent","germination_rate","avg_germinative_energy_percent","germinative_energy_percent","avg_germinative_capacity","germinative_capacity","percent_germination_15degC_incubated","percent_germination_20degC_incubated"),
               names_to = "responseType",
               values_to = "responseValue")`

**Comment by wangxm-forest** [comment]
@dbuona 
Problem solved! I removed some weird empty rows much earlier in the cleaning code.

**Comment by dbuona** [comment]
@lizzieinvancouver I was making plots in budseedleadin.R for now

**Comment by lizzieinvancouver** [comment]
@wangxm-forest Do you think this is done? Can you possibly link to some plots and related issues and close this?

**Comment by wangxm-forest** [comment]
We made some plots on chill duration and response variables, see [here](https://github.com/lizzieinvancouver/egret/commit/70bb7138ec414a45e302ef71957c0b54bc9d5557#diff-df398fec69a7a148a5906af57752ac0e097adbbaeabe162fd0fa30c1db6f6645)
